test/typescript/styling-comparison.spec.tsx

```diff
@@ -68,11 +68,11 @@ const DecoratedUnionProps = decorate<ArtProps>( // <-- without the type argument
       const props = this.props;
       return (
         <Typography classes={props.classes}>
-          {props.category === 'book' ? props.author : props.artist}
+          {props.category === "book" ? props.author : props.artist}
```

Comment:
Please revert.

Reply:
Ok, done, but what is the intended preference?
Both are used in that file, roughly 50/50

Reply:
My immediate preference is to keep irrelevant changes out of this PR 🙂